metadata.json

```
@@ -13,8 +13,8 @@
       "version_requirement": ">= 9.0.0 < 10.0.0"
     },
     {
-      "name": "WhatsARanjit/node_manager",
-      "version_requirement": "0.7.5"
+      "name": "puppetlabs/node_manager",
+      "version_requirement": ">= 1.0.1"
```

Please add an upper bound to ensure we don't pull in breaking changes
```suggestion
      "version_requirement": ">= 1.0.1 < 2.0.0"
```